src/builder/webpack/config.node.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import { join, basename } from "path"
-import { BannerPlugin } from "webpack"
```

Comment:
Why not `import { BannerPlugin, optimize: { UglifyJsPlugin } } from "webpack"` ?